.github/workflows/testsuite.yml

```diff
@@ -145,14 +151,14 @@ jobs:
           ./Configure -des -Dusedevel ${{ matrix.CONFIGURE_ARGS }} -Dprefix="$HOME/perl-blead" -DDEBUGGING
       - name: Build
         run: |
-          MALLOC_PERTURB_=254 MALLOC_CHECK_=3 make -j2
+          MALLOC_PERTURB_=254 MALLOC_CHECK_=3 make -j2 test_prep
       - name: Show Config
         run: |
           LD_LIBRARY_PATH=. ./perl -Ilib -V
           LD_LIBRARY_PATH=. ./perl -Ilib -e 'use Config; print Config::config_sh'
       - name: Run Tests
         run: |
-          MALLOC_PERTURB_=254 MALLOC_CHECK_=3 TEST_JOBS=2 make -j2 test_harness
+          LD_LIBRARY_PATH=`pwd` MALLOC_PERTURB_=254 MALLOC_CHECK_=3 TEST_JOBS=2 ./perl t/harness
```

Comment:
Why "`pwd`" here rather than "." as above? Should the above be changed too? (And similarly for the FIXUP commit.)

Reply:
"Because it changes directory, so the lazy approach used previously doesn't work"...

Yes, good catch. I added a commit in the rebase that changes all those `.`s to ```pwd```s

Thanks.